src/modules/dashboard/components/Dialogs/AwardAndSmiteDialogs/ManageReputationDialogForm/ManageReputationDialogForm.css

```diff
@@ -108,3 +109,50 @@
   width: 100%;
   background-color: color-mod(var(--temp-grey-blue-7) alpha(15%));
 }
+
+@media screen and query700 {
+  .percentageSign {
+    position: static;
+  }
+
+  .inputText {
+    margin-top: 0;
+    width: clamp(55px, 15%, 104px);
+    min-width: fit-content;
+    float: none;
+    transform: translateY(-20px);
+  }
+
+  .inputContainer {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-end;
+    width: auto;
+    column-gap: 6px;
+  }
+
+  .inputContainer label {
+    margin-top: 0px;
+    position: static;
+  }
+
+  .inputContainer label span {
+    display: inline-block;
+    min-height: 34px;
+  }
+
+  .inputContainer input {
+    padding-right: 0px;
+  }
+
+  .inputContainer > div:first-child {
+    flex: 1 0 80%;
+    min-height: 95px;
+  }
+
+  .inputContainer > div:nth-child(2) {
+    flex: 1 0 10%;
+    align-self: flex-end;
+    transform: translateY(-26px);
```

Comment:
Correct me if wrong but I think the line above messed up the alignment between the error message and the Max reputation msg?

<img width="390" alt="Screen Shot 2022-07-19 at 10 53 30 AM" src="https://user-images.githubusercontent.com/71563622/179699716-be653f09-ebbc-4f2b-82a3-a8590de7e258.png">